README.md

```diff
@@ -329,21 +329,28 @@ All discounts are added as one `CustomLineItem` with a negative price. This item
 
 ## Free shipping
 
-Free shipping is one of our discount codes type. You can define it on two ways. Firstly you can create predefined coupon with `Free shipping` type. This coupon is connected to pre created product with `source id = 5h1pp1ng`. 
-The second way is to create new or use existing product which will represent Your shipping method.
-If you have chosen your product now You can create new discount with unit type of this product. When you apply whichever of this discount information about it will be set to commerce tools custom field named shippingProductSourceIds.
-Next step is to properly define shipping method in Your commerce tools panel and configure [Predicates](https://docs.commercetools.com/tutorials/shipping-method-with-predicate). Go to `Settings -> Project settings -> Shipping methods`. Use existed or create new shipping method which will be applied if one of codes will be used.
+Free shipping is one of our discount codes type. To handle this case You must define Free shipping code. You can define it on two ways:
+
+1. You can create predefined coupon with `Free shipping` type.
+   This coupon is connected to pre created product with `source id = 5h1pp1ng`.
+   In this case You can clearly define coupon type by this ID because it's given by Voucherify and cannot be changed.
+2. To create new or use existing product which will represent Your shipping method.
+   If you have chosen your product now You can create new discount with unit type of this product.
+
+When you apply whichever of this discount code, the connected `product id` it will be set to commercetools cart custom field named shippingProductSourceIds.
+Next step is to properly define shipping method in Your commercetools panel and configure [Predicates](https://docs.commercetools.com/tutorials/shipping-method-with-predicate). Go to `Settings -> Project settings -> Shipping methods`. Use existed or create new shipping method which will be applied if one of codes will be used.
 In `Shipping method -> Predicate` field You can define condition when a given shipping method will be available. To allow uses to use Your new free shipping method you need to define formula.
 
-`custom.shippingProductSourceIds contains any ("5h1pp1ng")` - this formula is used for default free shipping code
-`custom.shippingProductSourceIds contains any ("<your_source_id>")` - this formula should be used when you want to apply this shipping method with custom vourcherify shipping method. 
+`custom.`shippingProductSourceIds` contains any ("5h1pp1ng")` - this formula is used for default free shipping code with predefined `source id = 5h1pp1ng`
```

Comment:
shouldn't be `custom.shippingProductSourceIds contains any ("5h1pp1ng")` (without two ` brackets)?